.github/workflows/featurenet-update.yml

```diff
@@ -46,6 +46,11 @@ on:
         description: 'Use short session aleph-node binary'
         required: true
         type: boolean
+      sudo-account-id:
```

Comment:
this MUST be the same as during creation, otherwise there will be problems, right?

Reply:
This is a temporary solution and we do have an item which we're working right now to remove all now 5 arguments from `update FE` workflow, namely
* expiration,
* short session checkbox,
* internal checkbox,
* number of validators,
* sudo account id
so that user do not need to retype those. To answer the question - in this case, it won't be a problem since bootstrap chain is not done in `update FE` workflow, but maybe I'll remove it anyway to not cause any confusion.